notebooks/utils.py

```diff
@@ -0,0 +1,97 @@
+import json
+
+import pandas as pd
+import requests
+from schemas.jobs import JobResultDownloadResponse
+
+MODEL_INFO_FILE = "assets/model_info.csv"
+
+EVAL_METRICS = {
+    "Meteor": ["meteor", "meteor_mean"],
+    "BERT Precision": ["bertscore", "precision_mean"],
+    "BERT Recall": ["bertscore", "recall_mean"],
+    "BERT F1": ["bertscore", "f1_mean"],
+    "ROUGE-1": ["rouge", "rouge1_mean"],
+    "ROUGE-2": ["rouge", "rouge2_mean"],
+    "ROUGE-L": ["rouge", "rougeL_mean"],
+}
+
+
+def _parse_model_results(result: dict, mi: pd.DataFrame):
```

Comment:
could we add a more descriptive var for `mi` (i.e. are these model inferences?)

Reply:
`model_info` but yes, you're right. I'll change the var name.